src/text/line.rs

```diff
@@ -568,6 +568,53 @@ impl<'a> Line<'a> {
     pub fn push_span<T: Into<Span<'a>>>(&mut self, span: T) {
         self.spans.push(span.into());
     }
+
+    /// An internal implementation method for `WidgetRef::render_ref`
+    ///
+    /// Allows the parent widget to define a default alignment, to be
+    /// used if `Line::alignment` is `None`.
+    pub(crate) fn render_with_alignment(
```

Comment:
nit: move this below where it's used so that code is easier to read top to bottom.

Reply:
Good call, and thank you for the change.  I had been grouping it with the other methods in the `impl Line` block, but on double-checking, those are all publicly-exposed methods.

Reply:
I'm tempted to make a rust style lint for method ordering - it's a strong preference of mine that methods should be ordered from abstract to specific like this generally, but I understand that the opposite preference also exists (and so it's worth having both / making that configurable). Having it as a lint would save having to express and repeat the preference as a contributing thing, which would be nice.